src/ocm/components/hosts/UpdateDay2ApiVipModal.tsx

```diff
@@ -9,7 +9,7 @@ type UpdateDay2ApiVipModalProps = {
 const UpdateDay2ApiVipModal: React.FC<UpdateDay2ApiVipModalProps> = ({ isOpen, ...props }) => (
   <Modal
     aria-label="Update API IP Dialog"
```

Comment:
We should probably rename the `aria-label` too to be consistent with the title.

Reply:
Changed it